docs/package.json

```diff
@@ -31,14 +31,14 @@
     "@fortawesome/fontawesome-svg-core": "^6.2.1",
     "@fortawesome/free-solid-svg-icons": "^6.2.1",
     "@fortawesome/react-fontawesome": "^0.2.0",
-    "@mui/base": "5.0.0-alpha.126",
+    "@mui/base": "^5.0.0-alpha.127",
```

Comment:
Unstable packages should have exact versions (without `^`)

Reply:
Fixed ! 🤦